src/main/java/uk/gov/ons/fwmt/job_service_v2/config/QueueConfig.java

```diff
@@ -85,8 +85,7 @@ public SimpleMessageListenerContainer container(ConnectionFactory connectionFact
 
   @Bean
   public MessageListenerAdapter listenerAdapter(JobServiceMessageReceiver receiver) {
-    MessageListenerAdapter listenerAdapter = new MessageListenerAdapter(receiver, "receiveMessage");
-    return listenerAdapter;
+    return new MessageListenerAdapter(receiver, "receiveMessage");
```

Comment:
comment for retryTemplate below, and the 5000, 3.0 and 45000 values be moved as new keyvalues in the yaml
rabbitmq:
  initialinterval: 5000
  multiplier: 3.0
  maxInterval: 45000